base_layer/wallet/src/transaction_service/protocols/transaction_receive_protocol.rs

```diff
@@ -269,9 +272,12 @@ where TBackend: TransactionBackend + Clone + 'static
                 );
             }
 
-            info!(
+            trace!(
```

Comment:
Seems like good debug-level information ?

Reply:
I was on the fence about this one because there is a message just above it with the tx_id which is what you would need to trace thing. The amount and message is not really useful for debug.